content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -123,10 +125,18 @@ spec:
 
 ### Class
 
-A volume snapshot can request a particular class by specifying the name of a
+A *volume snapshot* can request a particular class by specifying the name of a
 [VolumeSnapshotClass](/docs/concepts/storage/volume-snapshot-classes/)
 using the attribute `snapshotClassName`.
-Only VolumeSnapshotContents of the requested class, ones with the same `snapshotClassName`
-as the VolumeSnapshot, can be bound to the VolumeSnapshot.
+Only *volume snapshot contents* of the requested class, ones with the same `snapshotClassName`
+as the `VolumeSnapshot`, can be bound to that `VolumeSnapshot`.
 
 {{% /capture %}}
+
+## Provisioning Volumes from Snapshots
+
+A new volume, pre-populated with data from a snapshot, can be provisioned using *dataSource*
```

Comment:
"pre-populated" sounds like static binding which is not necessarily true here.  How about this?

A new volume can be provisioned using *dataSource* field in the `PersistentVolumeClaim` object to indicate the volume will be populated with data from a snapshot.

Reply:
I don't think it sounds like static binding (which we should stop calling static binding, and use the term pre-provisioned snapshots). Especially in the future when we introduce the notion of data populators.

Reply:
Ok, I see "pre-populated" is also used in our snapshot blog.  I'm okay with that.

Why should we stop calling static binding?  It is used for PV/PVC as well.

Reply:
We should update PVC docs too, but that's a separate issue.  I think "pre-existing" is a more descriptive word than "static".  If you say "bind to a pre-existing snapshot", I think that is more self-explanatory than "statically binding a snapshot".